src/scss/podcast-list.scss

```
@@ -1,41 +1,300 @@
+			&:hover {
+				filter: brightness(0.85);
+			}
+		}
+	}
+}
+
+/* Compact player for multi-column cards — mirrors castos-player.css @media (max-width: 768px) */
+.ssp-podcast-list--cards.ssp-podcast-list--col-2 {
+	.castos-player .player__main {
+		grid-template-columns: 60px 1fr;
+		grid-gap: 0;
```

Stylelint (17.5.0)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)